spec/dummy/db/schema.rb

```diff
@@ -182,7 +182,7 @@
     t.datetime "suspended_at"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.index ["email"], name: "email", unique: true, using: :btree
+    t.index ["email"], name: "email", using: :btree
```

Comment:
Oops...
Fixed https://github.com/cookpad/kuroko2/commit/0382965e21cb6cfc679fa8f8a88c47e1100fffc4
Please rebase or merge the master.